packages/svg-icons/scripts/svgoConfig.ts

```diff
@@ -1,6 +1,13 @@
 import type { Config } from "svgo";
 
-import { NeutralIconColor } from "./constants.ts";
+import { changeColorPlugin } from "./changeColorPlugin.ts";
+import { DecorativeOption7IconColor, DecorativeOption7SurfaceColor, NeutralIconColor, White } from "./constants.ts";
+
+export const colors: { [key: string]: string } = {
```

Comment:
export const colors = {
    [White]: "var(--hop-richicon-placeholder-neutral-surface, #fff)",
    [DecorativeOption7IconColor]: "var(--hop-richicon-placeholder-background, #2A2620)",
    [DecorativeOption7SurfaceColor]: "var(--hop-richicon-placeholder-foreground, #E5DED6)"
} satisfies Record<string, string>;

would be a better typing. 

Typescript sandbox to see the difference https://www.typescriptlang.org/play/?#code/FAUwHgDg9gTgLgAgMZQHYGdEBEQpgQzgEsA3EAeQmLQHYBJFVAYSgBtYEBeBAIgGIATAEEBANgEAGHgG4EAejkIAtEoAWUCEoAmuWIVIglG6qhpKijUJFiJGmBDjz6ylEzQDKAVxgAzfEhAWdhguXj4AUQBWLHCsURl5RRV1TR0nYjIjKiJaJXRvPwCraHhkNHsAdVUiOBBQ-h9GhIVlNQ08uEJPdCVUEE84AlY8gv8QYGKbMoxbNlh0UIBvYARVhABtKpqQAF0ALl4SfBgACmT2mAtqxiUIVjH1Vh0YXv7B-GH83zGAGgQ+Ro+ACUPB+KzW60cegyFGytAYaCCsH2h2OZzamkuSGuaFu9wCj2eSgARv4ANYAcxgUE8qC0f0EInEEhBYLWGyhBBhrhypi83wCSJgKJ4R1O50xVwsuLuDzYRJ8sBAVJpdIZURicRBwAAvgh0Pp0D4iCAFgAlXQwLQAHkwl1QFL+dqIDoAfNIJuASrMZmVgugBAdFhsySAAJ4HZ0OlFRikIPXcZbszbVWoisXolJKLE41B4uVPEAvPoDIYjAUgBmA1ng1aQy3OWFuBHMObCg6itES7NSm6ygnyosk8kq2n0-7CMSSGvJzmNnm0fmFQJt9NdjE97HSvP9kCEoeKmDK6lj9XRWKibU6j3AOBhiB1IXoADS4dCobDUB8CDvD6-fvmD1f0fNsA1fMN33Df9gP-FB-QEaQgA